src/desktop/components/light_detector/index.coffee

```diff
@@ -1,4 +1,4 @@
-{ BackgroundCheck } = require('background-check/background-check.js')
+BackgroundCheck = require('background-check/background-check.js')
```

Comment:
@orta noticed this issue, this is why BG check has been breaking! I think @melnguyen will be happy to see this fix.

Reply:
Oh oops! Didn't see we already opened https://github.com/artsy/force/pull/2624. Nice one @erikdstock

Reply:
Thanks! Yeah, this PR also does it - https://github.com/artsy/force/pull/2624